puppet/zulip/files/cron.d/update-analytics-counts

```diff
@@ -0,0 +1,5 @@
+MAILTO=root
+
+# Run update analytics tables every 5 minutes past the hour
+5 * * * *   zulip cd /home/zulip/analytics/management/commands && python manage.py update_analytics_counts
```

Comment:
We'll need to figure out what the right arguments to pass here are; I don't think the default arguments will work for ensuring that all time is up-to-date.